x/compute/internal/keeper/msg_dispatcher.go

```diff
@@ -186,6 +188,20 @@ func redactError(err error) (bool, error) {
 func (d MessageDispatcher) DispatchSubmessages(ctx sdk.Context, contractAddr sdk.AccAddress, ibcPort string, msgs []v1wasmTypes.SubMsg, ogTx []byte, ogSigInfo wasmTypes.VerificationInfo, ogCosmosMessageVersion wasmTypes.CosmosMsgVersion) ([]byte, error) {
 	var rsp []byte
 	for _, msg := range msgs {
+
+		if d.keeper.GetLastMsgMarkerContainer().GetMarker() == true {
+			// todo: break with error? probably
+			return nil, sdkerrors.Wrap(sdkerrors.ErrLastTx, "Cannot send messages or submessages after last tx marker was set")
+		}
+
+		if msg.Msg.LastMsgMark != nil {
+			//
+			d.keeper.GetLastMsgMarkerContainer().SetMarker(true)
+
+			// no handler is defined for marker - it's just to get here
```

Comment:
If you look a few lines down, the dispatcher sends the message according to it's type to a handler. This is here to let future us know that no handler exists for this message, and if we let it pass through it will error out